parser-typechecker/src/Unison/Codebase/BranchDiff.hs

```diff
@@ -98,22 +98,24 @@ computeSlices oldTerms newTerms oldTypes newTypes = (termsOut, typesOut)
       let nc = allNames oldTerms newTerms
           nu = allNamespaceUpdates oldTerms newTerms
        in DiffSlice
-            nu
-            (allAdds nc nu)
-            (allRemoves nc nu)
-            (remainingNameChanges nc)
-            (addedMetadata oldTerms newTerms)
-            (removedMetadata oldTerms newTerms)
+            { tallnamespaceUpdates = nu,
```

Comment:
No change here, just making these `DiffSlice` constructions easier to read by including the record field names.